php/Terminus/Instrument.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Terminus;
+use \ReflectionClass;
+use \Terminus\Request;
+use \Terminus\Collections\Bindings;
+
+class Instrument {
+  private $_id;
+  private $_attributes;
+  private $_bindings;
```

Comment:
An instrument doesn't need a binding relation. But I've been trying to add a `collection` property that maps back to the collection.